src/demo/index.html

```diff
@@ -408,9 +420,33 @@ <h1>Symphony Electron API Demo</h1>
             window.postMessage({ method, data }, '*');
         };
 
+        const getRTCLogs = () => {
+            if (document.getElementById('rtc-log-filename2').value == '') {
+                const filename = document.getElementById('rtc-log-filename').value;
+                const contents = document.getElementById('rtc-log-contents').value;
+                return [ { filename, contents }, ];
+            } else {
+                const filename1 = document.getElementById('rtc-log-filename').value;
+                const contents1 = document.getElementById('rtc-log-contents').value;
+                const filename2 = document.getElementById('rtc-log-filename2').value;
+                const contents2 = document.getElementById('rtc-log-contents2').value;
+                return [ { filename: filename1, contents: contents1 }, 
+                    { filename: filename2, contents: contents2 }, ];
+            }
+        }
+
         window.addEventListener('message', (event) => {
             const { data, method } = event.data;
             switch (method) {
+                case 'collect-logs':
+                    if( data === 'RTC' ) {
+                        if (window.ssf) {
+                            window.ssf.logReceiver('RTC', getRTCLogs());
+                        } else {
+                            postMessage( apiCmds.logReceiver, { logName: 'RTC', logFiles: getRTCLogs() } );
```

Comment:
I think it is less confusing as an example to just hardcode the string 'log-receiver' here, instead of having to look up a constant somewhere. It's only going to be used in one place anyway.

Reply:
Yeah, I'll change it to just the string

Reply:
If we decide to change the name in the future, it'd be hard to refactor. I think it'll be better if we have the string in a variable itself.

Reply:
@VishwasShashidhar This is a demo, which other people are supposed to look at to know how to use the API, right? So it should be as simple as possible. It should only include exactly what is needed to demonstrate the feature. No unnecessary details.

Reply:
While I agree in this use case @johankwarnmarksymphony, we should still from a refactoring POV, have a variable. In any case, like you said, this is the demo page, so, we can proceed with the current implementation.